astropy/table/tests/test_mixin.py

```diff
@@ -102,14 +104,41 @@ def test_io_ascii_write():
             t.write(out, format=fmt['Format'])
 
 
-def test_io_write_fail(mixin_cols):
+def test_io_Quantity_write(tmpdir):
     """
-    Test that table with mixin column cannot be written by io.votable,
-    io.fits, and io.misc.hdf5
-    every pure Python writer.  No validation of the output is done,
+    Test that table with Quantity mixin column can be written by io.votable,
+    io.fits, and io.misc.hdf5. No validation of the output is done,
```

Comment:
Per my "Review comment", I recommend just removing all the changes related to votable and HDF5 from this PR.